elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -27,6 +27,9 @@ window.PLRTE = function(uuid, options) {
     quill.setContents(quill.clipboard.convert(contents));
     
     quill.on('text-change', function(_delta, _oldDelta, _source) {
-        inputElement.val(btoa(unescape(encodeURIComponent(quill.root.innerHTML))));
+        inputElement.val(btoa(he.encode(quill.root.innerHTML, {
+            allowUnsafeSymbols: true, // HTML tags should be kept
```

Comment:
This sounds bad, but I assume it's actually ok?

Reply:
This means characters like `<`, `>`, `"`, etc. are not converted, which is what I actually want in this case.

Reply:
https://github.com/mathiasbynens/he#allowunsafesymbols

> The default value for the allowUnsafeSymbols option is false. This means that characters that are unsafe for use in HTML content (&, <, >, ", ', and `) will be encoded. When set to true, only non-ASCII characters will be encoded.

Reply:
Ok, cool, thanks for explanation!